### ISSUE TITLE
Training on Head CT and Brain Tumor dataset

### ISSUE THREAD

**Issue by nono-zz** [rohban-lab/Knowledge_Distillation_AD#10]
Hey, thanks for sharing the code, it really helps. Could you please provide code to train the model on Head and Brain dataset as well? Thank you very much.

**Comment by SMSD75** [comment]
Hi, 

Thank you for your interest in our research. Everything has been mentioned in the paper. You only should download the dataset and follow the instructions.

**Comment by nono-zz** [comment]
Hey, thanks for your reply, could you please tell me how you assign the test set from Brain Tumor and Head CT dataset?

I've downloaded these 2 datasets. The test set is composed of all of the anomalous samples and 20 normal samples, and the rest of the normals samples form the training set. I followed the same procedure as I reproduce the MVTec results, since there seems to be no specific instruction for these 2 medical datasets.

However, I didn't get the results as expected, the roc_auc value on BrainMRI and Head_CT are 0.8173 and 0.6080 respectively, I'm confused about the possible cause of my results, hope you could help me out with this!

I set the learning rate = 0.001, batch size = 64, img_size = 128, lama = 0.01.

**Comment by SMSD75** [comment]
You should randomly split between the train and test samples several times. Also, for a quick suggestion, change the image size to 256. I will try to check the problem.

**Comment by nono-zz** [comment]
Okay, I tried splitting randomly and training for several times, the results are very close to the paper. Thank you very much!
